testsuite/gnat2goto/tests/loop_array_range/test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL gnat2goto fails with contraint_error
```

Comment:
Can we have a comment in the test explaining why this test fails (if you know)?

Reply:
Fixed.